lib/cli/commands/create.ts

```diff
@@ -28,6 +28,10 @@ const formatOutput = (response: CreateNodeResponse.AsObject) => {
       walletInitializedMessage += response.initializedLndsList.join(', ');
     }
 
+    if (response.initializedConnext) {
+      walletInitializedMessage += ', ETH';
```

Comment:
- [ ] here is may be the situation when lnd clients already initialized (for example, when native xud is used). 
it is not common but still the case. In this case BTC and LTC will be missed in the string and as a result we will get ', ETH'. it would be good to handle this case.

Reply:
Ah very good catch, sorry for this I'll handle it!